docs/faq/proof-of-stake/general.md

```diff
@@ -1,16 +1,37 @@
-# <img class="dcr-icon" src="/img/dcr-icons/TicketLive.svg" /> PoS Voting General FAQ
+# <img class="dcr-icon" src="/img/dcr-icons/TicketLive.svg" /> Proof-of-Stake General FAQ
 
 ---
 
-#### 1. Do I need to be constantly connected to the network to participate in PoS?
+#### 1. When did Decred's Proof-of-Stake start?
 
-A wallet needs to be online 24/7 to respond to vote if one of your tickets is selected. There are two main ways to do this: 1) a solo staking wallet which you set up and keep online all the time, or 2) you can use a [Voting Service Provider](../../proof-of-stake/how-to-stake.md#pos-using-a-voting-service-provider-vsp) (VSP). A VSP is a community-run wallet to which you can assign the voting rights of your tickets. The VSP will vote on your behalf, charging a small percentage of the PoS reward as a fee for this service.
+Proof-of-Stake (PoS) was included in the first release of Decred software in February 2016.
 
-It is important to note that you are only assigning voting rights to the VSP, not your actual funds. A VSP does not have access to your funds at any time.
+Ticket purchases were possible as soon as the intial premine coins became
```

Comment:
airdrop coins

Reply:
As I understand, all of the premine coins (superset of the airdrop coins) matured at the same time, and any of those premine coins could have been used to purchase tickets?

The non-airdrop premine coins **weren't** used to buy tickets, because of verbal commitments, but technically they **could have** been?

Reply:
Perhaps just initial coins then.

I don't like the word premine because it's not really accurate.  There were no really any premined coins.  There were initial coins in block 1 that the miners had to choose to mine and extend.  Premine coins are realistically coins that were created in the genesis block, which they weren't.

Reply:
Happy to go with "initial coins". Pushed an update